ydb/library/yql/providers/yt/provider/yql_yt_dq_hybrid.cpp

```diff
@@ -79,6 +79,15 @@ class TYtDqHybridTransformer : public TOptimizeTransformerBase {
         return !HasSettingsExcept(*operation.Ref().Child(4U), DqOpSupportedSettings);
     }
 
+    bool HasDescOrderOutput(const TYtOutputOpBase& operation) const {
+        TYqlRowSpecInfo outRowSpec(operation.Output().Item(0).RowSpec());
+        const auto& sortDirections = outRowSpec.SortDirections;
+        if (std::any_of(sortDirections.cbegin(), sortDirections.cend(), [](const bool s) {return !s;})) {
+            return true;
+        }
+        return false;
```

Comment:
IsSorted() && HasAuxColumns()

Reply:
Replace to `return outRowSpec.IsSorted() && outRowSpec.HasAuxColumns()`